Lusas_Adapter/CRUD/Create/Loads/BarDistributedLoad.cs

```diff
@@ -53,19 +53,22 @@ public partial class LusasV17Adapter
                 );
 
             List<double> valuesAtA = new List<double> {
-                    barDistributedLoad.ForceA.X, barDistributedLoad.ForceA.Y,barDistributedLoad.ForceA.Z,
-                    barDistributedLoad.MomentA.X, barDistributedLoad.MomentA.Y,barDistributedLoad.MomentA.Z
+                    barDistributedLoad.ForceAtStart.X, barDistributedLoad.ForceAtStart.Y,barDistributedLoad.ForceAtStart.Z,
+                    barDistributedLoad.MomentAtStart.X, barDistributedLoad.MomentAtStart.Y,barDistributedLoad.MomentAtStart.Z
                 };
 
             List<double> valuesAtB = new List<double> {
-                    barDistributedLoad.ForceB.X, barDistributedLoad.ForceB.Y,barDistributedLoad.ForceB.Z,
-                    barDistributedLoad.MomentB.X, barDistributedLoad.MomentB.Y,barDistributedLoad.MomentB.Z
+                    barDistributedLoad.ForceAtEnd.X, barDistributedLoad.ForceAtEnd.Y,barDistributedLoad.ForceAtEnd.Z,
+                    barDistributedLoad.MomentAtEnd.X, barDistributedLoad.MomentAtEnd.Y,barDistributedLoad.MomentAtEnd.Z
                 };
 
             List<string> keys = new List<string>() { "FX", "FY", "FZ", "MX", "MY", "MZ" };
 
             List<int> ids = new List<int>();
 
+            string positioning = barDistributedLoad.RelativePositions ? "parametric" : "actual";
+            string axis = barDistributedLoad.Axis == LoadAxis.Global ? "global" : "local";
```

Comment:
```suggestion
            string axis;
            if(barDistributedLoad.Projected)
                        axis = "projected";
             else
                         axis = barDistributedLoad.Axis == LoadAxis.Global ? "global" : "local";
```

To account for projected loads.